test/solidityExecutionFramework.h

```diff
@@ -46,7 +46,20 @@ class ExecutionFramework
 	bytes const& compileAndRun(std::string const& _sourceCode, u256 const& _value = 0, std::string const& _contractName = "")
 	{
 		dev::solidity::CompilerStack compiler;
-		compiler.compile(_sourceCode, m_optimize);
+		try
+		{
+			compiler.compile(_sourceCode, m_optimize);
+		}
+		catch (const std::exception& e)
```

Comment:
did not see that in the coding standard. But by checking around the code we seem to always put underscore in the exception in catch so I will comply. Perhaps we should add a line in the coding standard? A quick look did not show anything.